CONTRIBUTING.md

```diff
@@ -39,29 +39,31 @@ remains the most accurate source for individual contributions.
 
 ## Code reviews
 
-All submissions, including submissions by project members, require review. We
+All submissions, including submissions by maintainers, require review. We
 use GitHub pull requests (PRs) for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
 
 ## Presubmits
 
-Several of our presubmit builds will only run automatically if you are a project
-collaborator. Otherwise a collaborator must label the PR with "kokoro:run". If
-you are sending code changes to the project, please ask to be added as a
+Most of our presubmit workflows will only run automatically on PRs if you are a
+project collaborator. Otherwise a maintainer must
+[approve workflow runs](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks).
+If you are sending code changes to the project, please ask to be added as a
 collaborator, so that these can run automatically. It is generally expected that
-PRs will only be merged when all presubmit checks are passing. In some cases,
-pre-existing failures may be ignored.
+PRs will only be merged when all checks are passing. In some cases, pre-existing
+failures may be ignored by a maintainer or admin.
```

Comment:
Only repo admins have this power, I believe.

Reply:
The pytype check is still non-blocking, right? That can be merged through by maintainers (write access) :P